README.md

```diff
@@ -1,2 +1,10 @@
 # qubes-fwupd
+
 fwupd wrappers for QubesOS
+
+## Testing
+
+```
+cd /usr/share/fwupd-qubes/
```

Comment:
I've included the installation process and usage in README.  

> For unit tests, it would be nice (at lest for some portion of them) to be usable not only in the Qubes.

It is already implemented. Tests that required Qubes skips, if you are testing in another OS. (I've also added a few words in README about this topic).

Reply:
Coll. Verified that.